lib/beaker/integrations/phoenix.ex

```diff
@@ -1,21 +1,40 @@
 defmodule Beaker.Integrations.Phoenix do
-  @moduledoc false
-  @behaviour Plug
-  import Plug.Conn, only: [register_before_send: 2]
+  @moduledoc """
+  A Plug for adding Beaker stats to your Phoenix pipeline(s)
+
+  plug Beaker.Integrations.Phoenix
 
-  def init(opts), do: opts
+  or
 
-  def call(conn, _opts) do
-    start_time = Beaker.Time.now
+  plug Beaker.Integrations.Phoenix, ignore_path: "my_special_path"
 
-    register_before_send(conn, fn(conn) ->
-      end_time = Beaker.Time.now
-      diff = end_time - start_time
+  ## Options
+    * `:ignore_path` - The top level of the URL to ignore, default is "beaker".
 
-      Beaker.TimeSeries.sample("Phoenix:ResponseTime", diff / 1000)
-      Beaker.Counter.incr("Phoenix:Requests")
+  """
+  @behaviour Plug
+  import Plug.Conn
+
+  def init(opts) do
+    Keyword.get(opts, :ignore_path, "beaker")
+  end
 
+  def call(conn, ignore_path) do
```

Comment:
Maybe we can pattern match this as two functions:
```
def call(conn = %Plug.Conn{path_info: [path | _]}, ignore_paths) when path in ignore_paths, do: conn
def call(conn, _ignore_paths) do
  ...
end
```

I'm not a big fan of branching logic inside of my functions :smile: